enos/ci/aws-nuke.yml

```diff
@@ -0,0 +1,426 @@
+- ap-southeast-1
+- ap-southeast-2
+- eu-central-1
+- us-east-1
+- us-east-2
+- us-west-1
+- us-west-2
+- global
+
+account-blocklist:
+ - 1234567890
+
+accounts:
+  # replaced in CI
+  ACCOUNT_NUM:
+    presets:
+      - default
+      - olderthan
+      - honeybee
+      - enos
+
+presets:
+  default:
+    # Ignores default VPC resources
+    filters:
+      EC2VPC:
+        - property: IsDefault
+          value: "true"
+      EC2RouteTable:
+        - property: DefaultVPC
+          value: "true"
+      EC2DHCPOption:
+        - property: DefaultVPC
+          value: "true"
+      EC2InternetGateway:
+        - property: DefaultVPC
+          value: "true"
+      EC2Subnet:
+        - property: DefaultVPC
+          value: "true"
+      EC2InternetGatewayAttachment:
+        - property: DefaultVPC
+          value: "true"
+  olderthan:
+    # Filters resources by age (when available)
+    # TIME_LIMIT replaced in CI
+    filters:
+      EC2Instance:
+        - property: LaunchTime
+          type: dateOlderThan
+          value: "TIME_LIMIT"
+      EC2NetworkACL:
+      EC2RouteTable:
+      EC2SecurityGroup:
+      EC2Subnet:
+      EC2Volume:
+      EC2VPC:
+        - property: tag:cloud-nuke-first-seen
+          type: dateOlderThan
+          value: "TIME_LIMIT"
+      ELBv2:
+        - property: tag:cloud-nuke-first-seen
+          type: dateOlderThan
+          value: "TIME_LIMIT"
+      ELBv2TargetGroup:
+      EC2NetworkInterface:
+      EC2InternetGateway:
+      EC2InternetGatewayAttachment:
+      RDSInstance:
+        - property: InstanceCreateTime
+          type: dateOlderThan
+          value: "TIME_LIMIT"
+
+  honeybee:
+    # Cloudsec
+    filters:
+      IAMRole:
+        - property: tag:hc-config-as-code
+          value: "honeybee"
+      IAMRolePolicy:
+        - property: tag:role:hc-config-as-code
+          value: "honeybee"
+      IAMRolePolicyAttachment:
+        - property: tag:role:hc-config-as-code
+          value: "honeybee"
+
+  enos:
+    # Existing CI to be cleaned up later
+    filters:
+      LambdaFunction:
+        - property: Name
+          value: "enos_cleanup"
+      IAMRole:
+        - property: Name
+          type: glob
+          value: "github_actions*"
+        - property: Name
+          type: glob
+          value: "enos*"
+        - property: Name
+          type: glob
+          value: "terraform-enos*"
+        - property: Name
+          type: glob
+          value: "boundary-*"
+        - property: Name
+          type: glob
+          value: "Enos*"
+        - property: Name
+          value: "rds-monitoring-role"
+      IAMRolePolicy:
+        - property: role:RoleName
+          type: glob
+          value: "enos*"
+        - property: role:RoleName
+          type: glob
+          value: "github_actions*"
+        - property: role:RoleName
+          type: glob
+          value: "terraform-enos*"
+        - property: role:RoleName
+          type: glob
+          value: "boundary-*"
```

Comment:
Is this correct, should it not be `vault-*`? How will this be when this is merged into `vault-enterprise`?